reverse-proxy/nginx.conf

```diff
@@ -0,0 +1,22 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
```

Comment:
for a setting with only one option you don't need the braces so i would rewrite 
`events  worker_connections 1024;`